BO4E-dotnet.Reporting/CompletenessReport.cs

```diff
@@ -263,17 +263,20 @@ public string ToCSV(string separator = ";", bool headerLine = true, string lineT
             if (Gaps.Any())
             {
                 DateTime minGap = this.Gaps.OrderBy(x => x.Startdatum).First().Startdatum;
-                DateTime maxGap = this.Gaps.OrderByDescending(x => x.Enddatum).First().Startdatum;
+                DateTime maxGap = this.Gaps.OrderByDescending(x => x.Enddatum).First().Enddatum;
                 columns.Add(minGap.ToString("yyyy-MM-ddTHH:mm:ssZ"));
                 columns.Add(maxGap.ToString("yyyy-MM-ddTHH:mm:ssZ"));
+                var gapsHours = (maxGap - minGap).TotalHours + 1;

```

Comment:
warum +1?
Entweder ist der Unittest falsch oder dein code :D

Reply:
Weil großer - kleiner macht nur diffrence inzwiechen 2 Anzahl und für richtige Antwort sollen wir ein mal hinzufügen.
z.b.: 4 - 2 = 2    =>    (2,3,4]  und
       4 - 2 + 1 = 3     =>    [2,3,4]

Reply:
aber wenn die gap 1 Stunde lang ist dann ist (maxGap-minGap).TotalHours=1 und damit die Anzahl fehlenderwerte = 1*4 = 4.

Reply:
vllt ist auch mein test falsch. wie groß ist denn im test die differenz in stunden?

Reply:
(01.07.2020 00:01:00 - 01.07.2020 00:00:00) glaube ich macht 0 Stunde und 59 Minuten.

Reply:
> (01.07.2020 00:01:00 - 01.07.2020 00:00:00) glaube ich macht 0 Stunde und 59 Minuten und 59 sekunden und 999 mili.

Sorry, nein ;)
```c#
[TestMethod]
public void TimeSpanDurationTest()
{
        Assert.AreEqual(1, (new DateTime(2020, 7, 1, 1, 0, 0, DateTimeKind.Utc) - new DateTime(2020, 7, 1, 0, 0, 0, DateTimeKind.Utc)).TotalHours);
}
```